### ISSUE TITLE
Ability to group time by second?

### ISSUE THREAD

**Issue by jongyonkim** [metabase/metabase#2680]
When will there be support for group time by second? The lowest granularity I see is hour.

**Comment by salsakran** [comment]
We actually discussed this at some length. The idea was to expose the granularities people had explicitly asked for (and actually written up in the SQL we audited on our hosted instances).

If you don't mind, could you give us some more context - 
How often will you be grouping by second? 
What are you using Metabase for at your company (business metrics, ops metrics, user behavior ,etc)?

**Comment by camsaul** [comment]
This would be pretty quick to implement if we decided we wanted it

**Comment by salsakran** [comment]
Yeah, biggest question is at what point does it overload what is already a pretty "full" tab.

**Comment by jongyonkim** [comment]
I have sensor data that varies by the second, and the goal is to visualize it vs time (without caring about any aggregate-based calculations).

I'm currently getting by using SQL, but it would be great if it is supported anyway.

**Comment by zbodi74** [comment]
From a customer:
> This would be critical for us since we have events that happen in seconds that we would like to keep track of. We are not necessarily looking to aggregate but the main intention is to create pivots or tables that will list the 'sequence' of events on a particular customer or lead that can be spaced seconds apart.
> This detail is used to set up automations, diagnose out of turn events etc which makes it a critical ask.
